deploy/charts/cert-manager/cainjector/templates/psp.yaml

```diff
@@ -0,0 +1,46 @@
+{{- if .Values.global.rbac.pspEnabled }}
+apiVersion: extensions/v1beta1
```

Comment:
Can you replace with `apps/v1`? (and throughout)

Reply:
@munnerz I don't get it. PSP are available in `extensions/v1beta1` or `policy/v1beta1`